src/vue/backend/src/views/GuidedSetupConnectApi.vue

```diff
@@ -0,0 +1,84 @@
+<template>
```

Comment:
Design flips into a mobile view at 1199w, making the design look awkward

![Screen Shot 2021-02-25 at 8 43 27 AM](https://user-images.githubusercontent.com/25048313/109187578-e185f200-7746-11eb-8ba0-3a3186a49360.png)